### PR TITLE
Add Cobalt Strike to the Kali AMI

### LINKED ISSUE #43: Cobaltstrike is missing from the Kali image
# 🐛 Bug Report

Cobaltstrike is not currently installed on the Kali images preventing the team from connecting to the teamserver from their Kali images.

## Expected behavior

Please install and license Cobaltstrike on the default Kali image.  Thanks.

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request adds [Cobalt Strike](https://cobaltstrike.com) to the software installed in the Kali image.

In the process of debugging why this AMI build was failing in GitHub Actions, I came across a few bugs/potential improvements in our existing workflows.  Thus a few commits in this PR have also been used to create pull requests in our skeleton repositories:
| Commit in this PR  | Corresponding skeleton PR |
| ------------- | ------------- |
| None | cisagov/skeleton-generic#59 |
| 953f7bc | cisagov/skeleton-packer#57 |
| dfd07fd | cisagov/skeleton-packer#58 |

(I did not make the cisagov/skeleton-generic#59 changes here because I didn't need them for the purposes of this pull request, but I did notice that they were possible and test them here.)

## 💭 Motivation and Context ##

@justinforbes requested this in #43.  This pull request resolves #43.

Note that this pull request also installs the Thunderbird mail client, which @justinforbes requested in [cisagov/cool-system#122](https://github.com/cisagov/cool-system/issues/122).

## 🧪 Testing ##

All pre-commit hooks pass.  I successfully built production and staging AMIs with these changes.  I deployed an instance to env1-staging using the new AMI and verified that Thunderbird and Cobalt Strike are both available.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.
* [x] New staging AMI built
* [x] New staging AMI tested
* [x] New production AMI built

### CODE REVIEW

**Comment by mcdonnnj** [comment]
*looks at the calendar* stahp
image

File: src/cobalt_strike.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  become: yes
+  become_method: sudo
+  roles:
+    - openjdk
+    - role: cobalt_strike
+      vars:
+        bucket_name: "{{ build_bucket }}"
+    - server_setup
+  tasks:
+    - name: Install thunderbird mail client
```
Comment (dav3r) [inline src/cobalt_strike.yml:13]: As long as we're fixing case issues here: `thunderbird` -> `Thunderbird`

**Review by dav3r — APPROVED** [review]
This all looks excellent- strong work! 🎳 
The only "fault" I could find was with a single character. 🔎

File: src/cobalt_strike.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          - dir_name: "CobaltStrike-Toolkit"
+            repo: |
+              https://github.com/killswitch-GUI/CobaltStrike-Toolkit/tarball/master
+          - dir_name: "Malleable-C2-Profiles"
+            repo: |
+              https://github.com/rsmudge/Malleable-C2-Profiles/tarball/master
+          - dir_name: "Malleable-C2-Randomizer"
+            repo: |
+              https://github.com/bluscreenofjeff/Malleable-C2-Randomizer/tarball/master
+    - name: Install mono for the manually installed packages that need it
```
Comment (mcdonnnj) [inline src/cobalt_strike.yml:57]:
```suggestion
    - name: Install Mono for the manually installed packages that need it
```

**Review by mcdonnnj — APPROVED** [review]
Strong work 💪

File: src/cobalt_strike.yml (hunk 2 above)
Comment (jsf9k) [reply src/cobalt_strike.yml]: See commit 8ae1be3.

File: src/cobalt_strike.yml (hunk 1 above)
Comment (jsf9k) [reply src/cobalt_strike.yml]: See commit f6ae79c.

**Pull request merged**